frontend/src/components/Scroll/ChevronScroll.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useCallback } from "react";
+import { IconChevronLeft, IconChevronRight } from "assets/images/images";
+import { combineClasses } from "components/Utility/utils";
+
+function ChevronScroll(props: React.PropsWithChildren<{}>) {
+  const [showRightChevron, setShowRightChevron] = React.useState(true);
+  const [showLeftChevron, setShowLeftChevron] = React.useState(false);
+  const [childIndex, setChildIndex] = React.useState(0);
+
+  const scrollRef = useRef<HTMLDivElement>(null);
+
+  const handleChevronVisibility = useCallback(
+    (e: React.UIEvent<HTMLDivElement>) => {
+      const { scrollLeft, scrollWidth, clientWidth } = e.currentTarget;
+      const maxScroll = scrollWidth - clientWidth - 1;
```

Comment:
Hey @MattPereira, can you help me understand why you subtracted 1 when calculating maxScroll?